### ISSUE TITLE
Android/iOS Splash Screen 

### ISSUE THREAD

**Issue by sam1rm** [u-guru/uguru#3874]
- [ ] Get this to work properly w/o the black / white screen flicker on Android
- [ ] The splashscreen should hide 2-3 seconds after the auto updating is complete  (to provide time for rendering)
- [ ] It should also fire on appResume for at least 3 seconds..
- [ ] Splashscreen should be turned on right before the window.location methods within the auto-update function in device service. Once 
- [ ] Splashscreen should be turned off after Device Ready
- [ ] Code should be general & also apply to iOS 

Basically the goal of the splashscreen is to prevent user from seeing any assets ever load one-by-one.
